workflow-templates/im-test-cypress.yml

```diff
@@ -0,0 +1,101 @@
+name: Run Cypress Tests
+
+# TODO:  In order to use this workflow a few changes need to be made to the cypress test configuration:
+# 1. Make the following npm package changes
+#    npm uninstall mocha-teamcity-reporter
+#    npm install --save-dev mochawesome mochawesome-merge mochawesome-report-generator
+# 2. Modify the cypress run section of the ci script by replacing the '--reporter ./mocha-teamcity-reporter' with '--reporter mochawesome --reporter-options overwrite=false,html=false,json=true'. For instance:
+#    old: "cy:ci" : "....&& cypress run --project ./ --reporter ./mocha-teamcity-reporter \"",
+#    new: "cy:ci" : "....&& cypress run --project ./ --reporter mochawesome --reporter-options overwrite=false,html=false,json=true \"",
+# 3. Add the following pre-ci script.  Your name will vary based on your cypress ci script name.  This removes results between test runs.
+#    "precy:ci": "rm -rf mochawesome-report/",
+# 4. Add the following script that merges all test json files and generates an html report.  Your name may vary based on your project conventions.  If the output file or results dir are changed, update the env values in the workflow.
+#    "cy:report": "mochawesome-merge mochawesome-report/*.json > merged-results.json && marge --reportDir cypress-results merged-results.json",
```

Comment:
Is there really a command called `marge` or is that a typo?

Reply:
It's really called marge,   'mochawesome report generator'

Reply:
😮

Reply:
I thought the same thing when the article I was reading had marge in the command.  I chuckled at their "typo".